coala_quickstart/green_mode/file_aggregator.py

```diff
@@ -0,0 +1,81 @@
+import os
+
+from coala_quickstart.green_mode.find_globs import (
+    find_globs_from_files,
+    )
+
+
+class Node:
+    """
+    Tree data-structure that has each directory as a node,
+    each subdirectory as children and files as leaves.
+    """
+
+    def __init__(self, name, parent, children=[]):
+        self.name = name
+        self.parent = parent
+        self.children = children
+
+    def add_children(self, children):
+        self.children.append(children)
+
+    def search_name(self, name):
+        if name == self.name:  # pragma: no cover
+            return self
+        for child in self.children:
+            if child.name == name:
+                return child
+        for child in self.children:
+            found_node = child.search_name(name)
+            if found_node is not None:  # pragma: no cover
+                return found_node
+        return None
+
+    def get_files(self, initials):
+        for child in self.children:
+            name = initials + os.sep + child.name
+            # FIXME: What happens in case of empty directories?
+            if child.children == []:
+                yield name
+            else:
+                yield from child.get_files(name)
+
+    def get_files_cd(self, initials):
```

Comment:
How is this function different from the previous one? add 1-2 lines of docstring

Reply:
It only returns files in the `cd` i.e. current directory to test for the glob `*` instead of the previous one which recursively looks for files in sub directories to test for the glob `**`